tools/docker/tick_manylinux1_x86_64/Dockerfile

```diff
@@ -7,8 +7,8 @@ ENV PATH="/root/.pyenv/bin:$PATH" SWIG_VER=3.0.12
 
 # Installing swig
 RUN curl -O https://kent.dl.sourceforge.net/project/swig/swig/swig-${SWIG_VER}/swig-${SWIG_VER}.tar.gz && tar -xf swig-${SWIG_VER}.tar.gz && \
-	cd swig-${SWIG_VER} && ./configure && make -j4 && make install && \
-	rm -rf swig-${SWIG_VER}.tar.gz swig-${SWIG_VER} 
+	cd swig-${SWIG_VER} && ./configure --without-pcre && make -j4 && make install && \
```

Comment:
What does this change :)?

Reply:
the base docker container that we build from is missing "libpcre"

so we now configure with ```--without-pcre```

Reply:
Sounds logical!